builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -738,16 +766,9 @@ def unDeployWebsite(stage) {
 	echo "unDeployWebsite::${service_config['service']}::${service_config['domain']} ${stage}"
 
 	withCredentials([
-		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+		[$class: 'AmazonWebServicesCredentialsBinding',  credentialsId: service_config.credentialId]
 	]) {
 		try {
-			def randomString = utilModule.generateRequestId();
-			def credsId = "jazz-${randomString}";
-			sh "aws configure set profile.${credsId}.region ${configLoader.AWS.REGION}"
-			sh "aws configure set profile.${credsId}.aws_access_key_id $AWS_ACCESS_KEY_ID"
-			sh "aws configure set profile.${credsId}.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"
-			sh "aws configure set preview.cloudfront true"
```

Comment:
This might be required to interact with cloudfront resources. Did the test this code flow? If this working, I'm assuming that aws cli got updated without the need of mentioning this but please validate this change.

Reply:
deletion of websites seems to be working in my tests. preview.cloudfront needs to be set to true when CloudFront needs to be invalidated.